packages/driver/src/cy/commands/querying.coffee

```diff
@@ -68,7 +70,10 @@ module.exports = (Commands, Cypress, cy, state, config) ->
 
     get: (selector, options = {}) ->
       ctx = @
-
+      ## can't get this error output any nicer as far as I know
+      if options is null or (typeIsArray options) or typeof options isnt 'object' then return $utils.throwErrByPath "get.invalid_options", {
```

Comment:
This is...sadly ideal. I don't totally understand the necessity behind the `typeIsArray` function. Why is `Array.isArray(options)` not sufficient?

Reply:
Also, you can use `_.isArray` here instead since lodash is already included: https://lodash.com/docs/4.17.15#isArray

Reply:
Ah, I see. Not that familar with lodash or coffescript so I just googled coffescript isArray check and that was the first result.